Deepanshu_Balani/project1 profile/profiles.css

```diff
@@ -0,0 +1,61 @@
+*{
+padding:0;
+margin:0;
+font-family: 'Gelasio', 'serif';
+}
+.back{
+ background-image: url("./my fav.jpeg");  
+ width:100%;
+ height:100vh;
+  background-size: cover;
+  display:flex;
+  align-items:center;  /*brings items in vertical center*/
+  justify-content:center;/*brings items in horizontal center*/
+  color:white;
+  text-align:center;/*keeps text aligned in center*/
+}
+
+.pro {
+   width:400px;
+  background-color:#120300;
+   opacity:0.7;/*increase the transparency*/
+   padding:40px 0;/*TB LR*/
+   border-radius:10%;/*used to curve the corners*/
+    
+}
+ .list{
+     list-style-position: inside;
+    font-size:1.0em;
+    font-weight:20;
+    border:solid white;
+    background-color:#120300 ;
+    opacity:0.7;
+    align-items: left;
+    justify-content: left;
```

Comment:
.list class is a flex container, if yes its is missing display:flex;
and in below lines, you are using align-items, justify-content. Will these 2  ever be in action?

Reply:
align items is removed ,justify content:space around is done .i cannot write display:flex .it will print list items in horizontal order.

Reply:
that's what I mentioned until you are making the element a flex container by using `display:flex;`, using flex container's properties will not have an effect on it.

Either use it or remove it's properties completely